internal/acceptance/tools/tools.go

```diff
@@ -47,22 +47,36 @@ func MakeNewPassword(oldPass string) string {
 
 // RandomString generates a string of given length, but random content.
 // All content will be within the ASCII graphic character set.
-// (Implementation from Even Shaw's contribution on
-// http://stackoverflow.com/questions/12771930/what-is-the-fastest-way-to-generate-a-long-random-string-in-go).
 func RandomString(prefix string, n int) string {
-	const alphanum = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
-	var bytes = make([]byte, n)
-	rand.Read(bytes)
-	for i, b := range bytes {
-		bytes[i] = alphanum[b%byte(len(alphanum))]
```

Comment:
I have changed how we generate random strings because this algorithm only works with ASCII chars: most non-ASCII chars take more than one byte to represent.